lib/index.js

```diff
@@ -148,46 +151,84 @@ function Log(x, y, vx, width){
   this.width = width;
 }
 
+function Turtle(x, y, vx, width){
+  this.x = x;
+  this.y = y;
+  this.vx = vx;
+  this.height = 50;
+  this.width = width;
+}
+
 Log.prototype.draw = function () {
   ctx.fillStyle = '#A0522D';
   ctx.fillRect(this.x, this.y, this.width, this.height);
   return this;
 };
 
+Turtle.prototype.draw = function () {
+  ctx.fillStyle = 'black';
+  ctx.fillRect(this.x, this.y, this.width, this.height);
+  return this;
+};
+
 Log.prototype.move = function () {
   this.x += this.vx;
   if(this.x > width + 50){
-    this.x = -50;
+    this.x = -200;
   return this;
   }
 };
 
+Turtle.prototype.move = function () {
+  this.x += this.vx;
+  if(this.x < -200){
+    this.x = 800;
+  return this;
+  }
+};
 
 var allCars = [];
 
-allCars.push(new CarLeft(50, 550, 1));
-allCars.push(new CarLeft(250, 550, 1));
-allCars.push(new CarLeft(450, 550, 1));
-allCars.push(new CarRight(600, 500, -1));
-allCars.push(new CarRight(400, 500, -1));
-allCars.push(new CarRight(50, 500, -1));
-allCars.push(new CarLeft(0, 450, 1));
-allCars.push(new CarLeft(100, 450, 1));
-allCars.push(new CarLeft(400, 450, 1));
-allCars.push(new CarRight(400, 400, -1));
-allCars.push(new CarRight(150, 400, -1));
-allCars.push(new CarLeft(300, 350, 1));
-allCars.push(new CarLeft(100, 350, 1));
+allCars.push(new CarLeft(50, 550, 1, 50));
+allCars.push(new CarLeft(250, 550, 1, 50));
+allCars.push(new CarLeft(450, 550, 1, 50));
+allCars.push(new CarRight(600, 500, -1, 50));
+allCars.push(new CarRight(400, 500, -1, 50));
+allCars.push(new CarRight(50, 500, -1, 50));
+allCars.push(new CarLeft(0, 450, 1, 50));
+allCars.push(new CarLeft(100, 450, 1, 50));
+allCars.push(new CarLeft(400, 450, 1, 50));
+allCars.push(new CarRight(400, 400, -1, 50));
+allCars.push(new CarRight(150, 400, -1, 50));
+allCars.push(new CarLeft(300, 350, 1, 50));
+allCars.push(new CarLeft(100, 350, 1, 50));
 
+//filler cars
+// allCars.push(new CarLeft(50, 550, 1, 50));
+// allCars.push(new CarLeft(250, 550, 1, 50));
+// allCars.push(new CarLeft(450, 550, 1, 50));
 
 var allLogs = [];
 
-allLogs.push(new Log(-50, 250, 1, 100));
-allLogs.push(new Log(-250, 250, 1, 100));
-allLogs.push(new Log(-450, 150, 1, 150));
-allLogs.push(new Log(-550, 150, 1, 150));
+allLogs.push(new Log(-200, 200, 1, 100));
+allLogs.push(new Log(50, 200, 1, 100));
+allLogs.push(new Log(450, 200, 1, 100));
+allLogs.push(new Log(350, 150, 2, 200));
+allLogs.push(new Log(0, 150, 2, 200));
+allLogs.push(new Log(-200, 50, 1.5, 150));
+allLogs.push(new Log(100, 50, 1.5, 150));
+allLogs.push(new Log(450, 50, 1.5, 150));
 
+var allTurtles = [];
```

Comment:
Cool @jksmall0631 . I think I have some turtle graphics we can use. I wonder if we can make them go underwater.